docs/JE-Makai81.md

```diff
@@ -0,0 +1,5 @@
+# JE-Makai81
+
+- **Name:** JE-Makai81
+- **Occupation:** GitHub newbie
+- **Location:** London
```

Comment:
What if you were a bit more specific here.  Are you talking about the London in the United Kingdom?  :gb: :smiley: